### PR TITLE
Couple of small fixes to the body

### PR DESCRIPTION
**Pull request by heftymouse**
Changed body margins, removed gradient (it looked out of place to me), made Dotsimus top bar text larger, added fancy invite and premium buttons (idrk about this, might remove it later)

### CODE REVIEW

**Comment by heftymouse** [comment]
Also I hid the Dotsimus text on mobile, I'll replace it with just the logo later

File: src/index.js [hunk 1]
```diff
@@ -44,11 +45,12 @@ document.addEventListener('DOMContentLoaded', function () {
...
 document.getElementById("app").innerHTML = `
 <header>
-    <div class="py-4 px-2 lg:mx-4 xl:mx-12 ">
+    <div class="py-4 px-2 lg:mx-4 xl:mx-12">
         <div class="">
             <nav class="flex items-center justify-between flex-wrap  ">
                 <div class="flex items-center flex-no-shrink text-white mr-6 ">
-                    <div class="text-sm lg:flex-grow mt-2 text-black title-font text-base">Dotsimus</div>
+                    <img class="items-center rounded-full h-14 w-14 ml-5 lg:ml-0 shadow-lg" src=${avatar} alt="Dotsimus Avatar"/>
+                    <div class="hidden lg:block lg:flex-grow pl-5 t-2 text-xl text-black font-bold title-font text-base">Dotsimus</div>
```
Comment (Jasius) [inline src/index.js:53]: It seems that there's a slight height misalignment since the addition of the avatar. 
image

**Review by Jasius — CHANGES_REQUESTED** [review]
Thank you for the changes, there's just one issue that I've noticed other than that I like the changes!

File: src/index.js (hunk 1 above)
Comment (heftymouse) [reply src/index.js:53]: Fixed!

Comment (heftymouse) [reply src/index.js:53]: It's a pretty hacky fix though, it seems to be having issues with the `xl` text size so I just added some extra padding on top

**Review by Jasius — APPROVED** [review]
Thank you once again for your help!!

**Pull request merged**